conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureSafeLongList.java

```diff
@@ -70,4 +81,24 @@ public void clear() {
     public SafeLong set(int index, SafeLong element) {
         return SafeLong.of(delegate.set(index, element.longValue()));
     }
+
+    static final class Serializer extends JsonSerializer<ConjureSafeLongList> {
+        @Override
+        public void serialize(ConjureSafeLongList val, JsonGenerator gen, SerializerProvider _serializer)
+                throws IOException {
+            // This is safe to serialize without checking because ConjureSafeLongList
+            // can only store SafeLong's
+            long[] longs = val.delegate.toArray();
+            gen.writeArray(longs, 0, longs.length);
+        }
+    }
+
+    static final class Deserializer extends JsonDeserializer<ConjureSafeLongList> {
+        @Override
+        public ConjureSafeLongList deserialize(JsonParser parser, DeserializationContext _ctxt) throws IOException {
+            // Avoid making a copy of the value from jackson
+            long[] longs = parser.readValueAs(long[].class);
+            return new ConjureSafeLongList(new LongArrayList(longs));
```

Comment:
Maybe fine, but I think this will move the failure of unsafe longs from parse time to access time (and maybe allow proxying of unsafe longs depending on the exact conjure setup)

Reply:
O ew, nah, it needs to be upfront. Will incorporate when I have a moment to address feedback.

Reply:
I will address this when I reintroduce deserialization see: https://github.com/palantir/conjure-java/pull/2386/files#r1813038405